src/app/shared/pipes/tyre-axle-load/tyre-axle-load.pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ReferenceDataTyre } from '@models/reference-data.model';
+
+@Pipe({
+  name: 'tyreAxleLoad'
+})
+export class TyreAxleLoadPipe implements PipeTransform {
+  transform(axleLoad: string | undefined, index: string | undefined, factor: number): undefined | string {
+    if (axleLoad) {
+      return axleLoad;
+    }
+    const calculatedValue = index ? +index * factor : undefined;
+    return calculatedValue?.toString();
```

Comment:
```suggestion
return axleLoad ?? index ? String(+index * factor) : undefined
```

Reply:
I'd have to bracket it like so:
```js
    return axleLoad ?? (index ? String(+index * factor) : undefined);
```
Doesn't look very nice.
Also I try to not "nest" ternary operators/ `??` `&&` `||`. Less readable and sonarqube usually flags it as a code smell.

I think a half-implementation of the suggestion probably is good though